src/model/behavioral/attribute/generator_attribute.py

```diff
@@ -291,10 +291,11 @@ def generate_attribute(self,agent,kd_map, logger, logging):
 			else:
 				counter -= prof["weight"]
 		self.counter+=1
+		'''
```

Comment:
Are you already logging proffession generation somewhere else? If you commented out this logging, maybe it is okay to remove this?